sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/MonitorCRUDTests.cs

```diff
@@ -0,0 +1,570 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Workloads.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Workloads.Tests.Tests
+{
+    [TestFixture]
+    public class MonitorCRUDTests : WorkloadsManagementTestBase
+    {
+        public MonitorCRUDTests() : base(true, RecordedTestMode.Record)
```

Comment:
```suggestion
        public MonitorCRUDTests(bool isAsync) : base(isAsync)
```
Reason for this change:

- We shouldn't force `isAsync` to equal to true, it will make only the asynchronous methods to be tested, but we also expose the API with synchronous methods. 
- I don't think we should set the `TestMode` here. For example, we will run test in Playback in the pipeline.